### PR TITLE
Phpunit wrapper

### PR DESCRIPTION
**Pull request by DavertMik**
* Use standalone package as a compatibility layer for PHPUnit. This will allow using different PHPUnit versions smoothly (with 7.0)
* Switched to use modern PHPUnit class names in source code

### CODE REVIEW

File: tests/cli/RunCest.php [hunk 1]
```diff
@@ -92,27 +92,16 @@ public function runXmlReportsInStrictMode(\CliGuy $I)
...
      * @group reports
      *
      * @param CliGuy $I
      */
     public function runCustomReport(\CliGuy $I)
     {
-        $I->wantTo('try the reporting mode');
+        if (\PHPUnit\Runner\Version::series() >= 7) {
+            throw new \Codeception\Exception\Skip('Not for PHPUnit 7');
+        }
```
Comment (Naktibalda) [inline tests/cli/RunCest.php]: Does this mean that PhpUnit 7 does not support custom reports?

**Comment by Naktibalda** [comment]
This build failed because of yii2-starter earlier, so I restarted it.
This time coverate tests failed with error `Class 'SebastianBergmann\CodeCoverage\CodeCoverage' not found`:
```
- RemoteServerWithPHPUnitCept: Try generate remote codecoverage phpunit reportPHP Fatal error:  Class 'SebastianBergmann\CodeCoverage\CodeCoverage' not found in /home/travis/build/Codeception/Codeception/src/Codeception/Coverage/Subscriber/Printer.php on line 36
```

**Comment by DavertMik** [comment]
Yeah, some issues still there. I will need to push the last fixes to make it pass

**Comment by DavertMik** [comment]
I think with this refactoring it's better to call new version 2.4.0 as it may introduce some unexpected breaking change...

**Comment by DavertMik** [comment]
@Naktibalda do you know why old PHPUnit is installed in https://github.com/Naktibalda/codeception-zf2-tests 

Install happens here https://travis-ci.org/Codeception/Codeception/jobs/343325829#L2059
And failure goes here https://travis-ci.org/Codeception/Codeception/jobs/343325829#L4023

**Comment by DavertMik** [comment]
It was zend-test. Please merge my PR https://github.com/Naktibalda/codeception-zf2-tests/pull/7

**Pull request merged**